test/components/tree-view.spec.tsx

```diff
@@ -123,18 +122,223 @@ describe('<TreeView />', () => {
             const onSelectItem = sinon.spy();
             const { select } = clientRenderer.render(<TreeView dataSource={treeData} onSelectItem={onSelectItem}/>);
 
-            const nodeLabelToClick = getTreeItem(treeData[0].label);
+            selectItemWithLabel(select, treeData[0].label);
+
+            return waitFor(() => expect(onSelectItem).to.have.been.calledWithMatch(treeData[0]));
+        });
+
+        describe('Keyboard Navigation', () => {
```

Comment:
it seems like there's A LOT of shared code between all the keyboard tests... should extract it to a method where possible